### DISCUSSION TITLE
CORS API

### DISCUSSION THREAD

**Discussion by lejibxl** [bluenviron/mediamtx#1451] [Questions]
is it possible to make a CORS for the API?

If the website is hosted on another server, it is possible to view the video (WebRTC) of the RPI as indicated in the documentation via an iframe, but to modify the parameters via the API seems impossible to me, the post commands must come from a cros.

Would it be possible to add something like APIAllowOrigin: '*' ?

**Comment by aler9** [comment]
Hello, APIs are not conceived to be used by a browser, but by a backend, anyway, I need to gather some documentation on how this issue is solved in other projects.

[a comment by an automated account is not shown]
